backend/src/main/java/com/github/chuettenrauch/mixifyapi/security/listener/OAuth2AuthenticationSuccessEventListener.java

```diff
@@ -1,49 +1,42 @@
 package com.github.chuettenrauch.mixifyapi.security.listener;
 
+import com.github.chuettenrauch.mixifyapi.security.exception.NoSuchOAuth2MapperException;
 import com.github.chuettenrauch.mixifyapi.security.mapper.OAuth2UserMapper;
+import com.github.chuettenrauch.mixifyapi.security.mapper.OAuth2UserMapperFactory;
 import com.github.chuettenrauch.mixifyapi.user.model.User;
 import com.github.chuettenrauch.mixifyapi.user.service.UserService;
+import lombok.RequiredArgsConstructor;
```

Comment:
Doesn't work for me because Lombok is optional in pom.xml

Reply:
was genau meinst du?

Reply:
Lösche diese Zeile. https://github.com/chuettenrauch/mixify/blob/GH-16-me-endpoint/backend/pom.xml#L55

Ich habe keinen Lombok global auf meinen Rechner installiert. Also wird es das hier von meinem Maven nicht runtergeladen. Dammit funktioniert dein Code nicht in meinem Local Dev Env.

Reply:
done